tests/ui/mut_key.rs

```diff
@@ -52,4 +61,16 @@ fn main() {
     tuples::<Key>(&mut HashMap::new());
     tuples::<()>(&mut HashMap::new());
     tuples_bad::<()>(&mut HashMap::new());
+
+    raw_ptr_is_ok(&mut HashMap::new());
+    raw_mut_ptr_is_ok(&mut HashMap::new());
+
+    // Currently fail as expected:
```

Comment:
This comment won't make sense in the future.

Reply:
It's gone :smiley:.